app/views/checkbox_criteria/_criterion_editor.html.erb

```diff
@@ -1,3 +1,5 @@
+<% marks_released = flash[:notice].present? %>
```

Comment:
Rather than define this here, compute it once in the main (top-level) view that is initially rendered and that includes this partial, and then pass the boolean value into the partial as a local variable. Please do this for all partials.